sockets/docker/docker.go

```diff
@@ -115,10 +123,9 @@ func (p *Poller) pollCurrentConnectionsFor(container docker.APIContainers, file
 	if err != nil {
 		return err
 	}
-	var stdout, stderr bytes.Buffer
+	var stdout bytes.Buffer
```

Comment:
hmm ideally it'd end up _somewhere_... but since this is a pretty narrow use case perhaps it's OK to discard it